awx/ui/client/features/templates/templatesList.controller.js

```diff
@@ -191,7 +191,7 @@ function ListTemplatesController(
     function refreshTemplates() {
         Wait('start');
         let path = GetBasePath('unified_job_templates');
-        qs.search(path, $state.params.template_search)
+        qs.search(path, $state.params.template_search, { 'X-WS-Session-Quiet': true })
```

Comment:
This line actually calls https://github.com/ansible/awx/blob/devel/awx/ui/client/src/shared/smart-search/queryset.service.js#L269 so I think we'll want to modify that function to take in an optional headers param instead of the `/lib/models/Base.js` search function which is something different